README.md

```diff
@@ -21,7 +21,9 @@ docker-compose up -d db
 ```
 
 ### Build the api container
-
+If `tmp/pids/server.pid` is missing from the root directory of the project, just
```

Comment:
why don't we add this directory to the git repo as we've done in https://github.com/sul-dlss/workflow-server-rails/commit/a81590f4a9005438e9afbf7380fae65d0a9b949f ?

Reply:
Done, new PR #103 follows same strategy .